src/main/target/SPEEDYBEEF7V3/target.h

```diff
@@ -23,152 +23,45 @@
-#define USE_TARGET_IMU_HARDWARE_DESCRIPTORS
-
-#define USE_SPI
-#define USE_SPI_DEVICE_1
-#define SPI1_SCK_PIN            PA5
-#define SPI1_MISO_PIN           PA6
-#define SPI1_MOSI_PIN           PA7
-
-#define USE_IMU_MPU6000
-#define IMU_MPU6000_ALIGN       CW270_DEG_FLIP
-#define MPU6000_SPI_BUS         BUS_SPI1
-#define MPU6000_CS_PIN          PB2
-#define MPU6000_EXTI_PIN        PC4
-
-#define USE_IMU_BMI270
-#define IMU_BMI270_ALIGN        CW0_DEG_FLIP
-#define BMI270_SPI_BUS          BUS_SPI1
-#define BMI270_CS_PIN           PC15
-#define BMI270_EXTI_PIN         PC3
-
-#define USE_EXTI
-#define USE_MPU_DATA_READY_SIGNAL
-
 // *************** I2C(Baro & I2C) **************************
 #define USE_I2C
 
 #define USE_I2C_DEVICE_1
-#define I2C1_SCL                PB8
-#define I2C1_SDA                PB9
+
 
 // Baro
 #define USE_BARO
 #define USE_BARO_BMP280
-#define USE_BARO_MS5611
-#define USE_BARO_DPS310
-#define USE_BARO_SPL06
+
 #define BARO_I2C_BUS          	BUS_I2C1	
 
 // Mag
 #define USE_MAG
-#define MAG_I2C_BUS             BUS_I2C1
-#define USE_MAG_HMC5883
-#define USE_MAG_QMC5883
-#define USE_MAG_IST8310
-#define USE_MAG_IST8308
-#define USE_MAG_MAG3110
-#define USE_MAG_LIS3MDL
-
-// *************** Internal SD card **************************
-
-#define USE_SPI_DEVICE_3
-#define SPI3_SCK_PIN            PC10
-#define SPI3_MISO_PIN   	    PC11
-#define SPI3_MOSI_PIN   	    PC12
-
-#define USE_SDCARD
-#define USE_SDCARD_SPI
-#define SDCARD_SPI_BUS          BUS_SPI3
-#define SDCARD_CS_PIN           PD2
 
-#define ENABLE_BLACKBOX_LOGGING_ON_SDCARD_BY_DEFAULT
 
 // *************** OSD *****************************
 #define USE_OSD
 #define USE_SPI_DEVICE_2
-#define SPI2_SCK_PIN            PB13
-#define SPI2_MISO_PIN           PB14
-#define SPI2_MOSI_PIN           PB15
-
-#define USE_MAX7456
-#define MAX7456_SPI_BUS         BUS_SPI2
-#define MAX7456_CS_PIN          PB12
-
-// *************** ADC *****************************
-#define USE_ADC
-#define ADC_CHANNEL_1_PIN           PC2
-#define ADC_CHANNEL_2_PIN           PC0
-#define ADC_CHANNEL_3_PIN           PC1
 
 #define VBAT_ADC_CHANNEL            ADC_CHN_1
 #define CURRENT_METER_ADC_CHANNEL   ADC_CHN_3
 #define RSSI_ADC_CHANNEL            ADC_CHN_2
 
 // *************** LED *****************************
 #define USE_LED_STRIP
-#define WS2811_PIN PC8
 
 // ********** Optiical Flow adn Lidar **************
 #define USE_RANGEFINDER
 #define USE_RANGEFINDER_MSP
 #define USE_OPFLOW
 #define USE_OPFLOW_MSP
 
+
 #define DEFAULT_FEATURES            (FEATURE_TX_PROF_SEL | FEATURE_CURRENT_METER | FEATURE_TELEMETRY | FEATURE_VBAT | FEATURE_OSD | FEATURE_BLACKBOX)
 #define USE_DSHOT
 #define USE_ESC_SENSOR
 #define USE_SERIAL_4WAY_BLHELI_INTERFACE
 
-#define SERIALRX_UART           SERIAL_PORT_USART2
-#define DEFAULT_RX_TYPE         RX_TYPE_SERIAL
-#define SERIALRX_PROVIDER       SERIALRX_SBUS
-
-#define MAX_PWM_OUTPUT_PORTS        8
-
-#define CURRENT_METER_SCALE         490
 
```

Comment:
Also, this file needs to go back to how it was.

Reply:
Thank you for your help.